poutyne/framework/callbacks/logger.py

```diff
@@ -123,13 +124,16 @@ def __init__(self, writer):
         super().__init__(batch_granularity=False)
         self.writer = writer
 
-    def _on_batch_end_write(self, batch, logs):
+    def on_batch_end(self, batch_number, logs):
+        self._on_batch_end_write(batch_number, logs)
```

Comment:
Why add a on_batch_end() implementation here ?

Reply:
This is a mistake. I was working on a implementation of a gradient tracker.

Reply:
af3815a